crates/api_models/src/payments.rs

```diff
@@ -510,6 +512,26 @@ impl From<&PaymentMethodData> for AdditionalPaymentData {
 #[derive(Debug, Clone, Eq, PartialEq, serde::Deserialize, serde::Serialize, ToSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum BankRedirectData {
+    BancontactCard {
+        /// The card number
```

Comment:
Can this struct be used for this? using `serde(flatten)`
https://github.com/juspay/hyperswitch/blob/6622cfa562db7e9f9f8caae974adc1a857f61a13/crates/api_models/src/payments.rs#L388

Reply:
@Narayanbhat166 We can but in future if we add any specific mandatory field to Card then it will impact here also. For that reason I kept it separately. And even now we have similar case where cvv is not required for BancontactCard. But in our Card it is mandatory